aviary/docs/getting_started/onboarding_level3.ipynb

```diff
@@ -340,13 +340,13 @@
     "    om.ExecComp(\n",
     "        [\n",
     "            \"takeoff_mass_con=takeoff_mass-climb_start_mass\",\n",
-    "            \"takeoff_range_con=takeoff_range-climb_start_range\",\n",
+    "            \"takeoff_distance_con=takeoff_range-climb_start_range\",\n",
     "            \"takeoff_vel_con=takeoff_vel-climb_start_vel\",\n",
     "            \"takeoff_alt_con=takeoff_alt-climb_start_alt\"\n",
     "        ],\n",
     "        takeoff_mass_con={'units': 'lbm'}, takeoff_mass={'units': 'lbm'},\n",
     "        climb_start_mass={'units': 'lbm'},\n",
-    "        takeoff_range_con={'units': 'ft'}, takeoff_range={'units': 'ft'},\n",
+    "        takeoff_distance_con={'units': 'ft'}, takeoff_range={'units': 'ft'},\n",
     "        climb_start_range={'units': 'ft'},\n",
```

Comment:
I noticed there is a "climb_start_range" that should probably also be changed to distance.

Reply:
for sure there are a bunch of other references to `range` in the code. I hesitate to do a global find/replace as I'm sure that will brick things. There are around 21 instances of `climb_start_range` I'll do a quick find/replace on those and see if all the tests are still working afterwards.